contracts/LifCrowdsale.sol

```diff
@@ -11,6 +11,10 @@ contract LifCrowdsale is Ownable, Pausable {
   // The token being sold
   LifToken public token;
 
+  // start and end of the public presale
+  uint256 public startPublicPresaleBlock;
+  uint256 public endPublicPresaleBlock;
```

Comment:
I think we should rename these variables as I said above, they should be:

`publicPresaleStartBlock`
`publicPresaleEndBlock`

Also, I think we might need a `privatePresaleEndBlock`, to see the finish date of the private presale. Or does it have to be exactly the same as `publicPresaleStartBlock`?